maps/site53/items/clothing/solgov-accessory.dm

```diff
@@ -83,12 +78,12 @@ badges
 	badge_string = "SCP Foundation"
 	slot_flags = SLOT_MASK | SLOT_TIE
 
-/obj/item/clothing/accessory/badge/solgov/representative
+/obj/item/clothing/accessory/badge/solgov/goc
```

Comment:
???
This should be /obj/item/clothing/accessory/badge/goc/representative

Reply:
fixed